functions/Get-DbaLogin.ps1

```diff
@@ -0,0 +1,135 @@
+
+.DESCRIPTION
+The Get-DbaLogin function returns an SMO Login object for the users passed, if there are no users passed it will return all logins.  
+
+.PARAMETER SqlInstance
+The SQL Server instance.You must have sysadmin access and server version must be SQL Server version 2000 or higher.
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. 
+
+.PARAMETER Logins 
+Pass a single login, or a list of them. Comma delimited. 
+
+.PARAMETER NoSystemLogins
+A switch to exlude system logins from returning. (Logins with '##' in the front or the 'sa' login)
+
+.PARAMETER Detail 
+Adds extra information by executing a TSQL Script against the server to get more information about the specified User. 
+Right now, this only adds LastLogin information
+
+.NOTES 
+Original Author: Mitchell Hamann (@SirCaptainMitch)
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.	
+
+.LINK
+https://dbatools.io/Get-DbaLogin
+
+.EXAMPLE
+Get-DbaLogin -SqlInstance SQl2016 
+Gets all the logins for a given SQL Server using NT authentication and returns the SMO login objects 
+
+.EXAMPLE   
+Get-DbaLogin -SqlInstance SQl2016 -SqlCredential $sqlcred 
+Gets all the logins for a given SQL Server using a passed credential object and returns the SMO login objects 
+
+.EXAMPLE 
+Get-DbaLogin -SqlInstance SQl2016 -SqlCredential $sqlcred -Logins dbatoolsuser,TheCaptain 
+Get specific user objects from the server
+
+.EXAMPLE 
+@('sql2016', 'sql2014') |  Get-DbaLogin -SqlCredential $sqlcred 
+Using Get-DbaLogin on the pipeline, you can also specify which names you would like with -Logins.
+
+.EXAMPLE 
+@( $LpsSql08) |  Get-DbaLogin -SqlCredential $sqlcred -detailed 
+Using Get-DbaLogin on the pipeline to get detailed information, like Last Login
+	
+#>
+	[CmdletBinding()]
+	Param (
+		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[object[]]$SqlInstance,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[Object[]]$Logins,
+		[Switch]$NoSystemLogins,
```

Comment:
Hmmmm, I don't like this type of switches very much. There are so many options. Now we have a -Disabled switch, but what if I only want to see the enabled ones? Or the 'Not locked', 'has no access', 'system', 'no system', ,...

I still have to know how to use `| where `, and it's actually not more work than to look up which switches are available.
Furthermore, there's a more complex logic needed, @SirCaptainMitch . What if multiple switches are supplied? Now only the first one is applied. That's not correct.
And you always have to be careful what switches exclude each other, and catch that too.

I'm in favour of dropping them all. @wsmelton ?